mmv1/products/blockchain/BlockchainNodes.yaml

```diff
@@ -0,0 +1,145 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BlockchainNodes'
+base_url: projects/{{project}}/locations/{{location}}/blockchainNodes
+self_link: projects/{{project}}/locations/{{location}}/blockchainNodes/{{name}}
+description: |
+  A representation of a blockchain node.
+
+
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/blockchain-node-engine'
+  api: 'https://cloud.google.com/blockchain-node-engine/docs/reference/rest/v1/projects.locations.blockchainNodes'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'firebase_project_basic'
+    min_version: 'beta'
+    primary_resource_id: 'default'
+    vars:
+      instance_name: 'memory-cache'
+      project_name: "my-project"
+    test_env_vars:
+      org_id: :ORG_ID
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  constants: templates/terraform/constants/firebase_project.go.erb
+  pre_create: templates/terraform/pre_create/firebase_project.go.erb
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The number of the google project that firebase is enabled on.
+  - !ruby/object:Api::Type::String
+    name: createTime
+    output: true
+    description: |
+      The timestamp at which the blockchain node was first created.
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    output: true
+    description: |
+      The timestamp at which the blockchain node was last updated.
+  - !ruby/object:Api::Type::Map
+    name: labels
+    description: |
+      User-provided key-value pairs
```

Comment:
Currently there's a compile error from this field, due to a missing required property for a `Map` type field. Instead of adding the property I think you should change labels to a `- !ruby/object:Api::Type::KeyValueLabels` type, which matches other labels fields elsewhere in this repo.